marklogic-data-hub-central/src/test/java/com/marklogic/hub/curation/controllers/FlowControllerTest.java

```diff
@@ -166,7 +167,7 @@ void getFlow() throws IOException {
     }
 
     @Test
-    void runFlow() {
+    void runFlow() throws IOException {
```

Comment:
Can you switch this to use the reference project instead of "run-flow-test"? I'd like for as many tests as possible to use the reference project. Adding a flow there with an entity-services-mapping would be a very useful addition.

Reply:
Used reference-project for the tests